Mathlib/MeasureTheory/Measure/MeasureSpace.lean

```diff
@@ -3559,6 +3559,20 @@ instance Add.sigmaFinite (μ ν : Measure α) [SigmaFinite μ] [SigmaFinite ν]
   refine' @sum.sigmaFinite _ _ _ _ _ (Bool.rec _ _) <;> simpa
 #align measure_theory.add.sigma_finite MeasureTheory.Add.sigmaFinite
 
+theorem SMul.sigmaFinite {μ : Measure α} [SigmaFinite μ] (c : ℝ≥0) :
+    MeasureTheory.SigmaFinite (c • μ) where
+  out' :=
+  ⟨{  set := spanningSets μ
+      set_mem := fun _ ↦ trivial
+      finite := by
+        intro i
+        simp only [smul_toOuterMeasure, OuterMeasure.coe_smul, Pi.smul_apply,
+          nnreal_smul_coe_apply]
+        refine Iff.mpr ENNReal.mul_lt_top_iff ?_
+        left
+        exact ⟨ENNReal.coe_lt_top, measure_spanningSets_lt_top μ i⟩
```

Comment:
```suggestion
        exact ENNReal.mul_lt_top_iff ENNReal.coe_lt_top (measure_spanningSets_lt_top μ i)
```

Reply:
You don't need the `iff` version and [ENNReal.mul_lt_top](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/Real/ENNReal.html#ENNReal.mul_lt_top) is easier to use.